integration/js/utils/AppTestStep.js

```diff
@@ -17,11 +17,13 @@ class AppTestStep extends TestStep {
       this.page = sessionInfo.page;
       this.driver = sessionInfo.driver;
     }
+
+    this.namespaceInfix = kiteBaseTest.payload?.namespaceInfix || '';
```

Comment:
Is this supposed to be the version number?

Reply:
Yes, `release e.x` or anything. just a way to differentiate the canary.

Reply:
Any chance we could make it a little more intuitive like `namespaceVersionInfix`? Just pretty hard to figure out what this value is actually supposed to be without context.